### ISSUE TITLE
swc not polyfill `string.replaceAll`

### ISSUE THREAD

**Issue by maoxiaoke** [swc-project/swc#2607]
### Describe the bug

i use `string.replaceAll`, while `replaceAll` not be polyfilled

### Input code

```typescript
'1234'.replaceAll('1', '2');
```

### Config

```json
{
  "env": {
    "targets": {
      "chrome": "49",
      "ie": "11",
    },
    "mode": "entry",
    "coreJs": "3.7"
  }
}
```

### Playground link

https://swc-play.vercel.app/?version=1.2.106&code=H4sIAAAAAAAAA1WNsQrCMBRF93zF3ZLCo5i0TlLBX3BwCR0ez6BCbMszGfx746bTGe65HOvDMNpe05ZZ0ilnZ70lhO5gjKzLq4BV%2Be0xIcYdwc%2BEGAjDlyNhP8%2FtfK2Szo%2FbvTgDOBapz5q5rEqQqpqWcuFcU4fpiJ%2B1bwXh4v4c09If9Cye%2BpYAAAA%3D&config=H4sIAAAAAAAAA0WMSw7CMAxE7%2BJ1trDIHTiEFdwqKD%2FZrkQU9e44NIjdaObNG%2FCSAH5AQxbimaQXxTd4oJBRAsem4AyzasMkdDpQ5J10InKzLdUqBF75IAc5lrj1KQo1NyaR9bMJy57opzFPrs9jFgO0N%2Fr67nD%2BHesX5bHA6%2FkBJNbjgrUAAAA%3D

### Expected behavior

`string.replaceAll` should be polyfilled

### Version

1.2.10

**Comment by zloirock** [comment]
https://github.com/swc-project/swc/issues/2209

**Comment by maoxiaoke** [comment]
@zloirock 
I already have imported `core-js/stable` in my entry point. And i found babel use [moudles-by-version](https://unpkg.com/browse/core-js-compat@3.19.0/modules-by-versions.json) to polyfill them. 

Means `corejs: '3.7'` is valid with `useBuiltIns: entry` in babel, while swc not.

I am thinking There must be some inconsistencies between swc and babel.

**Comment by zloirock** [comment]
You are right - `swc` uses a copy of [`modules-by-versions` from `core-js@3.6`](https://github.com/swc-project/swc/blob/d3944f520376b03e5df4939d85dfd3239ab7da58/ecmascript/preset-env/src/corejs3/modules-by-versions.json). In this case, it's this issue https://github.com/swc-project/swc/issues/1604

**Comment by Austaras** [comment]
swc only allows `coreJs: 3` but not `coreJs: "3.7"`

**Comment by zloirock** [comment]
@Austaras when I tested it (some months ago), it was allowed.

**Comment by Austaras** [comment]
It seems like type definition in swc playground is wrong

**Comment by zloirock** [comment]
https://github.com/swc-project/swc/issues/1606
